### ISSUE TITLE
Add postcss-import integration to docs?

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#613]
Ref: https://github.com/postcss/postcss-import/issues/117 & https://github.com/postcss/postcss-import/issues/145

Perhaps in the complementary tools doc?

**Comment by TrySound** [comment]
Both plugins will have the same options, so we can just add links and one example.

**Comment by jeddy3** [comment]
Sounds good to me :)

**Comment by TrySound** [comment]
Done. This way will process all imported files separately. Also this will prevent extra parsing and linting unused files.
```js
postcssImport({
  plugins: [
    stylelint()
  ]
});
```
https://github.com/postcss/postcss-import
https://github.com/TrySound/postcss-easy-import

**Comment by jeddy3** [comment]
Awesome. Thanks for this :)

**Comment by in-in** [comment]
How to use it with postcss-cli?
I get `stylelint is not a PostCSS plugin [postcss-import]`
```javascript
module.exports = {
  use: [
    'postcss-easy-import',
    'postcss-cssnext',
    'postcss-assets',
    'postcss-reporter',
  ],
  'dir': 'dist/assets/styles',
  'postcss-easy-import': {
    'glob': true,
    'plugins': ['stylelint']
  },
  'postcss-assets': {
    'basePath': 'dist/',
    'relative': 'assets/css',
    'loadPaths': ['assets/images/']
  },
  'postcss-cssnext': {
    'browsers': 'last 5 versions'
  },
  'stylelint': {
    'configFile': 'config/.stylelintrc'
  },
  'postcss-reporter': {
    'clearMessages': true
  }
};

```

**Comment by AlecRust** [comment]
Also wondering how to run this as a plugin of `postcss-import` using `postcss-cli`.

**Comment by jeddy3** [comment]
We recommend using stylelint as a [standalone lint tool](https://stylelint.io/user-guide/cli/). Use `stylelint` to lint your CSS and use `postcss-cli` transform your CSS.

**Comment by nebirhos** [comment]
For people having the same issue @in-in had: add stylelint as a module, not as a string.

```javascript
const stylelint = require('stylelint');

module.exports = {
  plugins: {
    'postcss-import': {
      plugins: [
        stylelint()
      ]
    },

  ...
}
```
